### PR TITLE
docs: add appium 2.0 driver cli docs

### PR DESCRIPTION
**Pull request by jlipps**
Add docs for the driver CLI functionality

### CODE REVIEW

File: docs/en/drivers/driver-cli.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+Note: this will update the driver(s) to the highest available minor version, but will *not* update to a new major version, since it might contain breaking changes. To update to a new major version, use the `--unsafe` flag.
+
+|Flag|Description|Default|
+|----|-----------|-------|
+|`--unsafe`|Upgrade to a new major version if available|None|
+
+|Example|Description|
+|-------|-----------|
+|`appium driver update xcuitest`|Update the XCUITest drivers to the latest safe version|
```
Comment (mykola-mokhnach) [inline docs/en/drivers/driver-cli.md:64]: I think it makes sense to add a possibility to be able to set the particular version number to install or to update to.

File: docs/en/drivers/driver-cli.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+## The Appium Driver Command Line Interface (CLI)
+
+Appium calls the bits of software that enable automation of a certain platform using a certain automation technology a "driver". There are drivers for iOS, Android, and other platforms. As of Appium 2.0, Appium does not download and install any drivers by default when you install Appium. There are too many drivers for this to be practical. Instead, you use a special command line interface to direct Appium to install and register drivers for you, to make them available for your use. This doc highlights the various commands available within this interface.
```
Comment (mykola-mokhnach) [inline docs/en/drivers/driver-cli.md:3]: It makes sense to mention that in the title, that we are talking about Appium 2.0+

**Review by mykola-mokhnach — APPROVED** [review]

File: docs/en/drivers/driver-cli.md (hunk 2 above)
Comment (jlipps) [reply docs/en/drivers/driver-cli.md:3]: this won't be published though until 2.0 is published, right, so at that point won't it be moot?

Comment (mykola-mokhnach) [reply docs/en/drivers/driver-cli.md:3]: publishing Appium2.0 does not mean that people would stop using 1.0 immediately

Comment (jlipps) [reply docs/en/drivers/driver-cli.md:3]: ok, fair enough!

**Comment by KazuCocoa** [comment]
👍

**Pull request merged**